### PR TITLE
feat: add dotfiles module

### LINKED ISSUE #5: Dotfiles
Basically we should replicate what we do for our dogfood template for dotfiles

- [x] coder_parameter
- [x] coder_script

### PR DESCRIPTION
**Pull request by bpmct**
closes #5

### CODE REVIEW

**Comment by matifali** [comment]
Are these ready to merge?

**Comment by bpmct** [comment]
Yeah, I suppose. Just need to test

**Review by matifali — APPROVED** [review]
LGTM. You can merge after testing

File: dotfiles/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+display_name: Dotfiles
+description: Allow developers to optionally bring their own dotfiles repository to customize their shell and IDE settings!
+icon: ../.icons/personalize.svg
```
Comment (matifali) [inline dotfiles/README.md:4]: Also, This icon is missing. Please add it.

**Review by matifali — CHANGES_REQUESTED** [review]

**Comment by matifali** [comment]
@bpmct can you get this in? It only needs a logo

**Review by matifali — APPROVED** [review]

**Pull request merged**